README.md

```diff
@@ -47,3 +47,9 @@ neuen CSV-Datei in target ausgegeben.
 Hiermit wird, mithilfe der Koordinaten, eine Weltkarte, inklusive markierter Strandorte, erstellt.
 
 Das shellscript führt anschließend Inkscape aus, um den relevanten Bereich vergrößert zu exportieren.
+
+#### for docker 
+#### build image
+docker build -t automate_image -f collaborators-worldmap/docker/Dockerfile .
```

Comment:
Use e more specific image name. "automate_image" could be used somewhere else and produce a conflict. E.g. 'godeep_collaborators_worldmap'.